util/bench/bench_runner.rs

```diff
@@ -0,0 +1,26 @@
+use std::process;
+
+fn main() {
+    let mut cmd = process::Command::new(env!("BENCH_BINARY"));
+
+    // Get args but skip the binary
+    let args: Vec<String> = std::env::args().skip(1).collect();
```

Comment:
In case args contain non-utf8

```suggestion
    let args: Vec<_> = std::env::args_os().skip(1).collect();
```